projects/addon-table/components/table-pagination/table-pagination.options.ts

```diff
@@ -7,6 +7,11 @@ function defaultSizeOptionContent({$implicit}: TuiContext<number>): string {
     return `${$implicit}`;
 }
 
+export interface TuiTablePagination {
+    readonly page: number;
+    readonly size: number;
+}
```

Comment:
This interface is not used anywhere inside `table-pagination.options.ts`. Should we return it back to `table-pagination.component.ts` ?

Reply:
fixed